data/remnant missions.txt

```diff
@@ -493,5 +592,771 @@ mission "Remnant: Technology Available"
+
+			label more
+			`	After examining <ship> for quite some time, she stops beneath it and looks up. "Is this a new form of ramscoop?" She asks as she gestures at your ship's Ramscoop. As you nod in the affirmative, she continues. "As I said before, fuel has always been a great limiter of our exploration," says the woman. "If you could get a couple extra that we could dismantle, this could be very useful to us."`
+			choice
+				`	"Sure, I could bring you some. What are you offering in return?"`
+				`	"Sorry, I'm really not interested in helping Remnant research."`
+					decline
+
+			`	After a bit of haggling, she agrees to pay you one million credits in exchange for a cargo of two Deep "Catalytic Ramscoops." Given how cheap they are to purchase in Deep space, you will be earning a very tidy profit on the deal.`
+				accept
+	
+	on complete
+		outfit "Catalytic Ramscoop" -2
+		payment 1000000
+		conversation
+			`You visit the local shipyard and eventually find the engineer busily repairing a damaged Starling. You inform her that you have brought a pair of "Catalytic Ramscoops" for her to study. She quickly tightens a couple more connectors and jumps down from her perch, gesturing at an assistant to transfer you the credits you were promised. "Thank you! Our maps of human space are old, and if they are still accurate, you traveled a very long way to bring these to us." Her chanting voice echoes off the starling above you with an interesting resonance.`
+			`	"Thank you for your help," she says. "If we can unlock the secrets of these scoops, it will give our ships greater range as they search for a safer refuge." As she picks up the remote control for the small loading truck with the two ramscoops she trills at you over her shoulder. "Oh, my name is Tali. I'll let you know what I find."`
+
+mission "Remnant: Plasma Cannons"
+	name "Retrieve Plasma Cannons for the Remnant"
+	description "An engineer on <planet>, a Remnant world, has offered to pay you one million credits in exchange for obtaining two Plasma Cannons."
+	minor
+	source Viminal
+	to offer
+		has "license: Remnant"
+		random < 50
+	on offer
+		require "Plasma Cannon"
+		conversation
+			`As you are browsing the spaceport shops, a man approaches you and introduces himself as a weapons specialist at one of the Remnant research facilities. He pauses hesitantly before beginning to chant: "As your ship is the only one we have seen in generations that has weapons from human space, we are intrigued by it. Could we take a closer look?"`
+			choice
+				`	"Yes, you are welcome to examine it."`
+					goto deep
+				`	"No, stay away from my ship."`
+					decline
+				`	"Yes, but don't touch anything."`
+
+			`	"Don't worry," he sings with a tinge of reproach. "I am a Remnant, not a Korath."`
```

Comment:
It is when they are being doubted by a stranger from ancestral space that they would like to make a good impression on. 

It is a line that I want the player to get regardless of which of the three missions they do, if they express doubt in the Remnant or over-protectiveness of their ship. If/When these missions are changed so that they can all be done instead of just one of the 3, that line would be removed from two of them.